public/javascripts/functionLibrary.js

```diff
@@ -7,6 +7,7 @@ var addSong = function(id) {
         }
     }).then(function(res) {
         res.url ? window.location.href = res.url : inLibrary(res.inLibrary)
+        console.log(res.inLibrary)
```

Comment:
still recommend renaming this file name to be more descriptive. function is a generic javascript concept. What is actually in this library? util.js is generic too, but it indicates this is like a kitchen sink. Unless you mean for all functions to go in this file.